firebase-functions/functions/src/operations/RoundOperations.ts

```diff
@@ -35,3 +46,24 @@ function getResult(player: QueryDocumentSnapshot<DocumentData>, allPlayers: Quer
       ? Result.Lost
       : Result.Won;
 }
+
+async function getRoom(roomId: string): Promise<Room> {
+  const roomDoc = await admin.firestore().collection(collections.rooms).doc(roomId).get();
+  return roomDoc.data() as Room;
+}
+
+function defineLastOneActive(playerChoices: PlayerChoice[], typeOfGame?: GameType): string | undefined {
+  return playerChoices.find(player => player.result === Result.Draw)
+    ? undefined
+    : typeOfGame === GameType.Loser
+      ? playerChoices.find(player => player.result === Result.Lost)?.playerId
+      : playerChoices.find(player => player.result === Result.Won)?.playerId;
+}
+
+function createNewGame(round: Round, typeOfGame?: GameType): Game {
```

Comment:
With the introduction of game, I don't think the filename is appropriate anymore. Also that whole notion of `persist` is not correct anymore I think, I used it back then when we thought this would only be for statistics, but now it is a real game concept.
Maybe `GameOperations` and `AddRoundToGame` or something?

Reply:
Here you go, you are right!